@xen-orchestra/web-core/lib/locales/en.json

```diff
@@ -106,7 +106,7 @@
   "core.pagination.all": "All",
   "core.pagination.show-by": "Show by",
   "core.query-search-bar.label": "Search Engine",
-  "core.query-search-bar.placeholder": "Write your query…",
+  "core.query-search-bar.placeholder": "Build your query…",
```

Comment:
I'm not sure it's a good idea to update a translated message, otherwise you'll also need to update `cs`, `de`, `es`, `fa`, `it`, `nl`, `ru` and `sv`.